src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -504,11 +786,11 @@ end
 		value = 1
 	})
 	```
-	
+
 	@param data PlayerEvent
-	@return { [any]: any }
+	@return Promise
```

Comment:
```suggestion
	@return Promise<T> 
```

Unless the type is already defined, you need to define custom return types with Moonwave by denoting them with a <T> after the type (Eg. `Promise<T>`)